src/main/java/com/netgrif/application/engine/workflow/domain/DataField.java

```diff
@@ -206,4 +207,19 @@ public int getMultiplicity() {
             throw new IllegalArgumentException("Variable arc must be an non negative integer");
         }
     }
+
+    @Override
+    public Referencable clone() {
+        DataField clone = new DataField();
+        clone.setBehavior(this.behavior);
```

Comment:
Is `DataField` ever cloned? I think `DataField` is only used in `Case` and does not need cloning. If cloning is needed then this function will create a shallow copy of its collections.